### ISSUE TITLE
Add ability to see Lirical and Exomiser results on the Family Page

### ISSUE THREAD

**Issue by lynnpais** [broadinstitute/seqr#3025]
**Is your feature request related to a problem? Please describe.**
We'd like to quickly review results

**Describe the solution you'd like**
Add ability to see Lirical and Exomiser results on the Family Page. Include rank number and disease association. Also include the ability to search for variants in these genes. 

**Describe alternatives you've considered**
NA

**Additional context**
Previously discussed

**Comment by hanars** [comment]
We will add a link to the family page under the "Show RNA-Seq OUTRIDER" link that says "Show Phenotype Prioritized Genes". When clicked it will open a modal which displays a table of all the genes we have phenotype prioritization scores for(we only load data for high-ranked genes). The table itself should have a first column column for "Gene", the display should use the inline gene detail display we use for SV genes that shows labels for omim/constraint/etc. We should also have a gene search link for this family/ gene to that column. The other columns in the table should share the display for the table we show on the hover for the label in search results. Table should be sorted by default by rank. Above the table should be a "Search for variants in all genes" link, similar to what we show for outrider.

**Comment by ShifaSZ** [comment]
> the display should use the inline gene detail display we use for SV genes that shows labels for omim/constraint/etc

I think that the inline gene detail display module is for all genes, not just SV ones. Is it right?

**Comment by hanars** [comment]
yes but we want the `showInlineDetails` to be true, which is only for SVs

**Comment by ShifaSZ** [comment]
> we only load data for high-ranked genes

How do we know a phenotype prioritized gene is high-ranked?

**Comment by hanars** [comment]
> > we only load data for high-ranked genes
> 
> How do we know a phenotype prioritized gene is high-ranked?

Ben only gives us high ranked genes in the lirical/exomiser output, we don't need to do any additional filtering for this

**Comment by ShifaSZ** [comment]
![image](https://user-images.githubusercontent.com/33550059/201169149-9b9e0b5a-0247-4a8e-be57-229fc776ced8.png)
@hanars Is this display what you mean?

**Comment by hanars** [comment]
no I want the display like on SVs, where it shows the labels next to the gene, although obviously without the "show gene expression". I also think it would be better if OMIM was just an orange label, not the box

<img width="344" alt="Screen Shot 2022-11-10 at 1 31 33 PM" src="https://user-images.githubusercontent.com/24598672/201177783-eb7b5637-27d5-486f-aa51-76c3594af8d8.png">

**Comment by lynnpais** [comment]
Looks good!
